### ISSUE TITLE
only admin users can access VIS page?

### ISSUE THREAD

**Issue by Schwarms** [ioBroker/ioBroker.admin#1792]
Can not access VIS page with other user than admin group users

**Steps to reproduce the behavior:**
1. Create a new user in a user group other than admin (in ioBroker admin interface)
2. add to this user group all priviliges / rights (in ioBroker admin interface)
3. make sure all widgets in VIS are enabled/allowed for this user group
4. enable authentification for web plugin
5. login as this new user (without admin rights) in the web login
6. open VIS runtime with this user

**Expected behavior**  
VIS should open and show the widgets.
Instead the VIS remains empty and the waiting glass shows up (turning forever in a loop).
Maybe this is important additional information: login with this new user in the iobroker admin interface is possible (admin interface is shown), but other interfaces like VIS, VIS-edit are not working (in VIS-edit an error message "No Connection" shows up).

**Versions:**  
 - Adapter version: VIS 1.4.15, WEB 5.2.10
 - Node version: V16.18.1
 - Operating system: Debian GNU / Linux 11

Maybe there is some misunderstaning on my side for the correct settings (setting missing?) but if that is the case I would like to request a feature-request for a more easy approach for this task.

Please see also here for more discussions to this issue, there are more users having problem with exact the same issue:
https://forum.iobroker.net/topic/54281/vis-views-benutzer-zugriffsberechtigung/

**Comment by Apollon77** [comment]
Should this not be a "web" issue rather than "admin"? The vis page is opened via the web port, right?

**Comment by Schwarms** [comment]
yes, sorry - I am new to this, can it be moved or do I have to reopen a new one in the web issues?
